server/views/pages/view/view.njk

```diff
@@ -28,56 +29,81 @@
+                {% elseif licence.typeCode === 'AP_PSS' %}
+                    <h1 class="govuk-heading-l">{{ actionWord }} licence and post sentence supervision order for {{ licence.forename }} {{ licence.surname }}</h1>
+                {% elseif licence.typeCode === 'PSS' %}
+                    <h1 class="govuk-heading-l">{{ actionWord }} post sentence supervision order for {{ licence.forename }} {{ licence.surname }}</h1>
+                {% else %}
+                    <h1 class="govuk-heading-l">{{ actionWord }} licence type {{ licence.typeCode }} for {{ licence.forename }} {{ licence.surname }}</h1>
+                {% endif %}
+            </div>
         </div>
-    </div>
-
-    <div class="govuk-grid-row">
-        <div class="govuk-grid-column-full">
-            {% if warningMessage %}
-                {{ govukWarningText({
-                   html: warningMessage
-                }) }}
-            {% endif %}
 
-            {{ inductionMeetingDetails(licence, isEditableByPrison, [], null, isPrisonUser) }}
-            {% if licence.typeCode === 'AP' or licence.typeCode === 'AP_PSS' %}
-                {{ additionalLicenceConditions(licence, additionalConditions) }}
-                {{ bespokeConditions(licence.id, null, null, null, licence.bespokeConditions) }}
-            {% endif %}
-            {% if licence.typeCode === 'PSS' or licence.typeCode === 'AP_PSS' %}
-                {{ additionalPssConditions(licence) }}
-            {% endif %}
-
-            <div class="govuk-button-group govuk-!-margin-top-8">
-                {% if isPrintable %}
-                    {{ govukButton({
-                        text: "View and print licence PDF",
-                        classes: "govuk-button--primary govuk-!-margin-right-1",
-                        href: '/licence/view/id/' + licence.id + '/pdf-print',
-                        attributes: { 'data-qa': 'print-licence', 'target': "_blank" }
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-full">
+                {% if warningMessage %}
+                    {{ govukWarningText({
+                    html: warningMessage
                     }) }}
                 {% endif %}
 
-                {{ govukButton({
-                   text: "Return to case list",
-                   classes: "govuk-button--secondary" if isPrintable,
-                   href: '/licence/view/cases',
-                   attributes: { 'data-qa': 'return-to-view-list' }
-                }) }}
+                {% if licence.kind === 'HARD_STOP' %}
+                    {{ hardStopInductionMeetingDetails(licence, isEditableByPrison, [], null, isPrisonUser) }}
+                {% else %}
+                    {{ inductionMeetingDetails(licence, isEditableByPrison, [], null, isPrisonUser) }}
+                {% endif %}
+
+                {% if licence.kind === 'HARD_STOP' %}
+                    <h2 class="govuk-heading-m">Licence conditions</h2>
```

Comment:
would it make sense for this text block to be a macro? think it might be used in a few places?